frontend/src/configs/config.ts

```diff
@@ -0,0 +1,33 @@
+import { config } from 'dotenv';
+
+interface AppConfig {
+  API_PATH: string;
+  SENTRY: {
+    DSN: string;
+    ENABLE: boolean;
+  };
+}
+
+export interface ConfigInterface {
+  APP: AppConfig;
+}
+
+const configuration = (): ConfigInterface => {
+  config();
+
+  const { VITE_API_ORIGIN_URL, VITE_SENTRY_DSN, SENTRY_ENABLE } = process.env;
```

Comment:
We **cannot** import env variables from the `process.env` in Vite. The variables must be imported from the `import.meta.env`

Reply:
Do we need additional config to import.meta.env. I mean path to .env file

Reply:
No, if the env file is in the same directory as the `package.json`. We don't even need to use the `dotenv` explicitly here, bc `vite` uses it under the hood already